includes/class-wcpdf-settings-callbacks.php

```diff
@@ -17,6 +17,18 @@ class Settings_Callbacks {
 	 */
 	public function section() {
 	}
+	
+	/**
+	 * Throw notice if image resolution is too high
+	 */
+	function check_image_resolution($resolution) {
+		if ($resolution > 600) {
+			echo 
+			'<div class="notice notice-info is-dismissible">
+				<p>The resolution of the image you uploaded for the Shop logo is above the recommended 600dpi!</p>
```

Comment:
@YordanSoares regarding the usage of `_e()`, take a look at this:
![Screenshot from 2021-11-02 11-00-14](https://user-images.githubusercontent.com/74280840/139894231-2aa78eff-96b9-4459-9b4d-dca146d57580.png)
source: https://developer.wordpress.org/reference/functions/_e/

So it looks like we should be using `esc_html( __())` instead..?

Reply:
I don't know how dangerous it can be not to escape the translation here. However, if necessary, you could use [`esc_attr_e()`](https://developer.wordpress.org/reference/functions/esc_attr_e/) instead. @Spreeuw what do you think?

Reply:
`_e()` is enough because it's a static text.

Reply:
@YordanSoares @alexmigf Technically it's static text, but since we don't have full control over the translations (they can be added by PTEs without our involvement), it's certainly not a bad idea to escape it. However, `esc_attr_e()` is specifically for HTML **attributes** (such as a `title` or `placeholder` attribute). The equivalent for HTML text is [`esc_html_e()`](https://developer.wordpress.org/reference/functions/esc_html_e/) (or [`esc_html__()`](https://developer.wordpress.org/reference/functions/esc_html__/) if you're not echoing directly)
Well spotted @dpeyou !